launchpod/angular-launchpod/backend/src/main/java/com/google/launchpod/data/Channel.java

```diff
@@ -51,4 +63,35 @@ public Channel(String name, String email, String podcastTitle, String descriptio
     this.language = language;
     this.description = description;
   }
+
+  public void setLanguage(String newLanguage) {
+    this.language = newLanguage;
+  }
+
+  public void setDescription(String newDescription) {
+    this.description = newDescription;
+  }
+
+  public String getLanguage() {
+    return this.language;
+  }
+
+  public String getDescription() {
+    return this.description;
+  }
+
+  public List<Item> getItems() {
+    return this.items;
+  }
```

Comment:
i actually don't see these being used anywhere. do we need them?

Reply:
I copied these over from Efrain's PR because we need these for the product alpha